nginx/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:16.04
```

Comment:
Nothing wrong with it, i just really like this one.  However you have a great point, i will remove it from this PR.

Thanks!